### PR TITLE
Added Hidden File Scanner ID

### PR DESCRIPTION
**Pull request by NamanArora**
With reference to this pr: https://github.com/zaproxy/zap-extensions/pull/1968 
#4585

### CODE REVIEW

**Comment by NamanArora** [comment]
@thc202 @kingthorin

File: docs/scanners.md [hunk 1]
```diff
@@ -157,6 +157,7 @@ Scanners:
 40032   .htaccess Scanner
 40033	NoSQL Injection MongoDB
 40034   .env File Scanner
+40039   Hidden File Scanner
```
Comment (kingthorin) [inline docs/scanners.md]: Is there a reason to skip to 39?

Comment (NamanArora) [reply docs/scanners.md]: Sorry. It was a typo from my previous pr. Corrected it now.

**Comment by thc202** [comment]
Thanks!

**Pull request merged**